### DISCUSSION TITLE
[Feature Request] Create new field layout tab when dragging a field onto "+ New Tab" button

### DISCUSSION THREAD

**Discussion by jasonkilp** [craftcms/cms#8640] [Ideas]
I think it would be helpful and make creating new sections more efficient if you could create a new field layout tab simply by dragging and dropping a field onto the "+ New Tab" button. It saves a little bit of time because you don't have to create a new tab first and then drag a field into it.

### Example of how it could look:
![craft-dragdrop](https://user-images.githubusercontent.com/5674532/53898154-86acda80-4005-11e9-996c-c3c53a1aee5f.gif)

**Comment by Mosnar** [comment]
As a bonus, I would love to be able to double-click a field to send it to the last tab rather than having to drag it up to the top and scroll back down.

**Comment by awrye** [comment]
Handy! I would definitely use this all the time.

+1
